Gopkg.toml

```diff
@@ -25,16 +25,17 @@
 #   unused-packages = true
 
 required = [
-    "github.com/grpc-ecosystem/grpc-gateway/runtime"
+    "github.com/grpc-ecosystem/grpc-gateway/runtime",
+    "github.com/coreos/bbolt"
 ]
 
-[[constraint]]
+[[override]]
```

Comment:
Why is this override? It's direct dependency for cn-infra: https://github.com/ligato/cn-infra/blob/master/rpc/grpc/plugin_impl_grpc.go#L25

Reply:
I recall @edwarnicke indicating there as a problem with cn-infra and newer grpc versions. The override here allows this to bump grpc up to version 1.14.0 (the latest). If this will be a problem, let me know.

Reply:
👍